import-service/src/libs/corsMiddleware.ts

```diff
@@ -0,0 +1,13 @@
+const addCorsToRequest = async (request) => {
+    if(request.response) {
+        request.response.headers = {
+            ...request.response.headers,
+            "Access-Control-Allow-Origin": "*"
```

Comment:
Does it make sense to create some constant `e.g. DEFAULT_HEADERS` and use it everywhere we need it (as I see there are at least 5 such places) in case we want to change it?

Reply:
Actually, it's just a little part of a huge issue caused by just copying the structure of product-service to import-service. I've put shared code of both services to a separate module in Task-6 solution